### ISSUE TITLE
HMIP device integration troubleshooting - Thermostats don't show data & Switches cannot be controlled

### ISSUE THREAD

**Issue by jwiesel** [SukramJ/hahomematic#656]
DONT'T DELETE THIS.

Please answer all questions as good as possible.

Language: english & german - both is fine

Make sure you have read the documentation at https://github.com/danielperna84/custom_homematic#custom_homematic. - done

Make sure you have read the release notes at https://github.com/danielperna84/custom_homematic/releases. - done 

Make sure you are running the latest version of the custom_component (and Home Assistant) before reporting an issue. - done

ENSURE that your Backend (CCU/Homegear/...) is working as expected and connected devices are controllable. - done

**custom_component version (if applicable):**
1.19.5

**known last working custom_component version (if applicable):**
???

**Home Assistant version (if applicable):**
2022.10.4

**CCU version:**
- Model (CCU2/3, Raspi3/4, NUC, ProxmoxVE, HA-Addon ...): Raspi 4
- OS (Original, RaspberryMatic, debmatic ...): RaspberryMatic
- Version: 3.65.11.20221005

**Problem-relevant configuration:**

Do you use tls? yes
Do you use callback? no
Do you use username and password? yes
Which interfaces do you use (hmip/bidcos/wired)? hmip + bidcos

**Describe the bug**
Connection to Homematic seems not to work completely.
I was able to connect to my RaspberryMatic and see als bidcos + hmip devices.
For BidCos Thermostats I can also see the temperature (HM-TC-IT-WM-W-EU).
For my HMIP thermostats (HmIP-WTH-2, HMIP-PS) I cannot see any data. I enabled "Actual Temperature" and "Humidity" in the advanced settings, but only see "Nicht verfügbar" / "not available" as value. for all Thermostats after several hours and restarts of the homematic integration in home assistant.
I tested switching on/off "HMIP-PS", that doesn't work either. See attached log.

Raspberrymatic should not be the root cause, as I have a running integration with iobroker, where all of the above is working.

**To Reproduce**
1. Go to 'Settings > Devices"
3. Select eq3 HMIP device
4. Enable "Actual Temperature" and "Humidity"
5. Value shows as "Not available" even after several minutes
5. Reload homematic (IP) Local integration
6. Value still not available

**Screenshots (if applicable)**
- CCU firewall
- integrations view
- device view
- enity view

**Relevant HA log entries (full log)**
```
Logger: hahomematic.client
Source: /usr/local/lib/python3.10/site-packages/hahomematic/client.py:399
First occurred: 09:50:03 (6 occurrences)
Last logged: 12:37:52

_set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000218A98ACC42:3, STATE, False
_set_value failed with ProxyException [(<Fault -1: 'Generic error'>,)]: 000218A995F0FB:3, STATE, False
_set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000218A98ACC42:3, STATE, True
_set_value failed with ProxyException [(<Fault -1: 'Generic error (TRANSACTION_DISCARDED_FOR_UNREACHABLE_DEVICE)'>,)]: 000218A995F0FB:3, STATE, True

```

**Diagnostic Information (available on every device)**
```
{
  "home_assistant": {
    "installation_type": "Home Assistant Container",
    "version": "2022.10.4",
    "dev": false,
    "hassio": false,
    "virtualenv": false,
    "python_version": "3.10.5",
    "docker": true,
    "arch": "x86_64",
    "timezone": "Europe/Berlin",
    "os_name": "Linux",
    "os_version": "5.10.0-17-amd64",
    "run_as_root": true
  },
  "custom_components": {
    "homematicip_local": {
      "version": "1.19.5",
      "requirements": [
        "hahomematic==2022.10.5"
      ]
    },
    "hacs": {
      "version": "1.28.0",
      "requirements": [
        "aiogithubapi>=22.2.4"
      ]
    }
  },
  "integration_manifest": {
    "domain": "homematicip_local",
    "name": "Homematic(IP) Local",
    "config_flow": true,
    "documentation": "https://github.com/danielperna84/custom_homematic",
    "issue_tracker": "https://github.com/danielperna84/hahomematic/issues",
    "requirements": [
      "hahomematic==2022.10.5"
    ],
    "ssdp": [
      {
        "manufacturer": "EQ3",
        "manufacturerURL": "http://www.homematic.com"
      }
    ],
    "zeroconf": [],
    "homekit": {},
    "dependencies": [],
    "codeowners": [
      "@danielperna84",
      "@SukramJ"
    ],
    "iot_class": "local_push",
    "loggers": [
      "hahomematic"
    ],
    "version": "1.19.5",
    "is_built_in": false
  },
  "data": {
    "config": {
      "entry_id": "3bc361bd9d5fe7fbd82f5615b13665c3",
      "version": 1,
      "domain": "homematicip_local",
      "title": "RaspberryMatic",
      "data": {
        "instance_name": "RaspberryMatic",
        "host": "192.168.178.100",
        "username": "**REDACTED**",
        "password": "**REDACTED**",
        "tls": true,
        "verify_tls": false,
        "callback_host": null,
        "callback_port": null,
        "json_port": null,
        "interface": {
          "HmIP-RF": {
            "port": 42010
          },
          "BidCos-RF": {
            "port": 42001
          }
        }
      },
      "options": {},
      "pref_disable_new_entities": false,
      "pref_disable_polling": false,
      "source": "ssdp",
      "unique_id": "D70992B5A5",
      "disabled_by": null
    },
    "platform_stats": {
      "binary_sensor": 19,
      "climate": 10,
      "cover": 10,
      "select": 1,
      "sensor": 45,
      "switch": 4
    },
    "devices": [
      "HM-CC-RT-DN",
      "HM-ES-PMSw1-Pl",
      "HM-ES-TX-WM",
      "HM-LC-Bl1PBU-FM",
      "HM-LC-Sw1-Pl-CT-R1",
      "HM-PB-2-WM55",
      "HM-Sec-SCo",
      "HM-TC-IT-WM-W-EU",
      "HMIP-PS",
      "HmIP-FALMOT-C12",
      "HmIP-KRC4",
      "HmIP-WTH-2"
    ]
  }
}

```

**Expected behavior**

HMIP Thermostats (HMIP-WTH-2) should show actual temperature & humidity
Switches should be controllable (HMIP-PS)

**Additional context**

**Comment by jwiesel** [comment]
![firewall](https://user-images.githubusercontent.com/11034083/196035475-a8d93693-d296-4dec-b39e-7347b09bdbba.png)

**Comment by jwiesel** [comment]
![device](https://user-images.githubusercontent.com/11034083/196035725-ba1c2199-e2c4-44dc-aa46-0df1d6582744.png)

**Comment by jwiesel** [comment]
![entities](https://user-images.githubusercontent.com/11034083/196035915-5761157e-3938-479e-8725-8ac92140e807.png)

**Comment by SukramJ** [comment]
Open the firewall.
Check the RM logs.
Is it possible to control your HmIP devices with CCU WebUI?

**Comment by SukramJ** [comment]
> Raspberrymatic should not be the root cause

I disagree:

These are errors reported back from your Raspberrymatic:
- `TIMEOUT`
- `TRANSACTION_DISCARDED_FOR_UNREACHABLE_DEVICE`

Do you have a high `CarrierSense Level` on RM?

**Comment by jwiesel** [comment]
Thanks for your swift response @SukramJ !
I just checked your comments and it seems like you are right.
Before I installed Homematic (IP) Local in Homeassistant this morning, I saw, that a RaspberryMatic update was available. I just installed that and went ahead with the homeassistant integration.
Right now it seems like my RaspberryMatic isn't fully working since the update..... (Learning: Never do two changes a the same time...)
I'll do further investigation and keep you posted after RaspberryMatic works again.
Thanks a lot!
